pxtarget.json

```
@@ -319,7 +319,6 @@
         "allowPackageExtensions": true,
         "homeScreenHero": "./static/hero.png",
         "socialOptions": {
-            "twitterHandle": "msmakecode",
             "orgTwitterHandle": "MSMakeCode",
             "hashtags": "MakeCode",
```

Maybe worth adding a hashtag to specify arcade in the tweet as well I suppose?

```suggestion
             "hashtags": "MakeCode Arcade",
```